tests/platform_tests/link_flap/test_link_flap.py

```diff
@@ -70,26 +70,25 @@ def test_link_flap(request, duthosts, rand_one_dut_hostname, tbinfo, fanouthosts
     logger.info("Orchagent CPU Util at end: %s", orch_cpu)
 
     # Record Redis Memory at end
-    end_time_redis_memory = duthost.shell(
-        r"redis-cli info memory | grep used_memory_human | sed -e 's/.*:\(.*\)M/\1/'")["stdout"]
-    logger.info("Redis Memory at start: %s M", start_time_redis_memory)
-    logger.info("Redis Memory at end: %s M", end_time_redis_memory)
+    end_time_redis_memory = get_avg_redis_mem_usage(duthost, 5, 5)
+    logging.info("Redis Memory at start: %f M", start_time_redis_memory)
+    logging.info("Redis Memory at end: %f M", end_time_redis_memory)
 
     # Calculate diff in Redis memory
-    incr_redis_memory = float(end_time_redis_memory) - float(start_time_redis_memory)
-    logger.info("Redis absolute  difference: %d", incr_redis_memory)
+    incr_redis_memory = end_time_redis_memory - start_time_redis_memory
+    logging.info("Redis absolute difference: %f", incr_redis_memory)
 
     # Check redis memory only if it is increased else default to pass
     if incr_redis_memory > 0.0:
-        percent_incr_redis_memory = (incr_redis_memory / float(start_time_redis_memory)) * 100
+        percent_incr_redis_memory = (incr_redis_memory / start_time_redis_memory) * 100
         logger.info("Redis Memory percentage Increase: %d", percent_incr_redis_memory)
-        incr_redis_memory_threshold = 10 if tbinfo["topo"]["type"] in ["m0", "mx"] else 5
+        incr_redis_memory_threshold = 15 if tbinfo["topo"]["type"] in ["m0", "mx"] else 10
```

Comment:
maybe it's worth putting this threshold into a function in link_flap_utils, so we dont have the constants across two seperate files.
or even just put the whole memory threshold comparison into its own function there, eg:
`def validate_redis_memory_increase(tbinfo, start_mem, end_mem)`